src/lib/FileHelper.ts

```diff
@@ -91,10 +96,11 @@ export async function pathExists(location: string): Promise<boolean> {
 export function findFiles(include: RelativePattern, maxResults?: number, errIfNoFound?: boolean) {
 	return new Observable<Uri>(observer => {
 		const tokenSource = new CancellationTokenSource();
-
+		let ignoreListFolders: string = getIgnoreList().join(',');
+		let ignoreGlob: string = `**/*{${ignoreListFolders}}`;
 		workspace.findFiles(
 			include,
-			undefined,
+			new RelativePattern(workspace.workspaceFolders![0], ignoreGlob),
```

Comment:
this may break cartridges detection since will look up only in first workspace

Reply:
Thanks for the finding - I will then adjust the methods to pass through the workspace param in another way and update the PR